### ISSUE TITLE
[Labels & Instructions – Make Code - Share] Visual label is not defined for 'Name for your project' edit field under 'Share Project' pop-up window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3592]
**User Experience**:
If visual label is not defined for “Edit Field" then screen reader user will not be able find out the information and this can cause inconvenience to the user, and ultimately lead to disorientation.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate through all the controls of  "Share Project" dialog.
5. Verify that visual label is defined for 'Name for your project' edit field by using developer tool(F12).

**Actual Result:**
Visual label is not defined for 'Name for your project' edit field under 'Share Project' pop-up window.

**Expected Result:**
Visual label should be define for 'Name for your project' edit field under 'Share Project' pop-up window.

**Note:**
Same issue appear under Chrome & Firefox.

**WCAG Reference:**
[WCAG 3.3.2 - Labels or Instructions
](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html})

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG3.3.2, #Win10-Edge(Chromium)

![WCAG3 3 2_Share project](https://user-images.githubusercontent.com/72871089/98222717-3541b500-1f77-11eb-90f7-0ac0fb3bfc9c.jpg)

**Comment by jwunderl** [comment]
Has placeholder / same behavior as name input in editor itself.
